pkg/server/token_minter.go

```diff
@@ -177,12 +177,25 @@ func (s *TokenMintServer) processRequest(r *http.Request) (int, string, error) {
 		return http.StatusForbidden, "requested permissions are not authorized for this repository", err
 	}
 
+	// The repository claim is of the form <org_name>/<repo_name>.
+	// Use this string split instead of attempting to use this and the repository_owner claim since
+	// the repository_owner claim is optional.
+	repoParts := strings.Split(claims.Repository, "/")
+	if len(repoParts) != 2 {
+		return http.StatusBadRequest, fmt.Sprintf("'repository' claim formatted incorrectly, requires <org_name>/<repo_name> format - received [%s]", claims.Repository), nil
+	}
+	// Lookup the App installation for the GitHub owner/repo
+	installation, err := s.githubApp.InstallationForRepo(ctx, repoParts[0], repoParts[1])
```

Comment:
I wonder what happens if the installation is in the org rather than the repo?

Reply:
The installation is at the org level, but when you configure it you either configure it for `all repositories` or specific individual repositories. There is no org without repos option AFAIK, I guess you could select no repos but then it would be useless.